.golangci.yaml

```diff
@@ -0,0 +1,9 @@
+linters:
+  # Disable all linters.
+  disable-all: true
+  # Enable specific linter
+  # https://golangci-lint.run/usage/linters/#enabled-by-default
+  enable:
```

Comment:
I wanted to enable them one at a time and fix per commit, otherwise things get slightly out of hand lol.

```
golangci-lint run
internal/template-validator/service/service.go:51:10: Error return value of `flag.Set` is not checked (errcheck)
        flag.Set("v", defaultServiceVerbosity)
                ^
internal/template-validator/service/service.go:52:10: Error return value of `flag.Set` is not checked (errcheck)
        flag.Set("logtostderr", "true")
                ^
internal/template-validator/service/service.go:63:16: Error return value of `f2.Value.Set` is not checked (errcheck)
                        f2.Value.Set(value)
                                    ^
internal/operands/common-instancetypes/reconcile.go:6:2: SA1019: "io/ioutil" has been deprecated since Go 1.19: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
        "io/ioutil"
        ^
internal/template-validator/virtinformers/vmcache.go:234:2: SA2001: empty critical section (staticcheck)
        v.lock.Unlock()
        ^
internal/operands/data-sources/reconcile_test.go:47:13: SA1019: fake.NewFakeClientWithScheme is deprecated: Please use NewClientBuilder instead. (staticcheck)
                client := fake.NewFakeClientWithScheme(common.Scheme)
                          ^
internal/template-bundle/bundle.go:7:2: SA1019: "io/ioutil" has been deprecated since Go 1.19: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
        "io/ioutil"
        ^
internal/template-validator/validation/eval.go:25:2: SA1019: "io/ioutil" has been deprecated since Go 1.19: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
        "io/ioutil"
        ^
internal/operands/metrics/reconcile_test.go:34:13: SA1019: fake.NewFakeClientWithScheme is deprecated: Please use NewClientBuilder instead. (staticcheck)
                client := fake.NewFakeClientWithScheme(common.Scheme)
                          ^
webhooks/ssp_webhook_test.go:49:39: Error return value of `sspv1beta1.SchemeBuilder.AddToScheme` is not checked (errcheck)
                sspv1beta1.SchemeBuilder.AddToScheme(scheme)
                                                    ^
webhooks/ssp_webhook_test.go:51:17: Error return value is not checked (errcheck)
                v1.AddToScheme(scheme)
                              ^
webhooks/ssp_webhook_test.go:209:6: func `checkExpectedError` is unused (unused)
func checkExpectedError(err error, shouldFail bool) {
     ^
webhooks/ssp_webhook_test.go:53:12: SA1019: fake.NewFakeClientWithScheme is deprecated: Please use NewClientBuilder instead. (staticcheck)
                client = fake.NewFakeClientWithScheme(scheme, objects...)
                         ^
tools/metricsdocs/metricsdocs.go:86:22: func `(*metricList).add` is unused (unused)
func (m *metricList) add(line string) {
                     ^
tools/metricsdocs/metricsdocs.go:89:13: SA1019: strings.Title has been deprecated since Go 1.18 and an alternative has been available since Go 1.0: The rule Title uses for word boundaries does not handle Unicode punctuation properly. Use golang.org/x/text/cases instead. (staticcheck)
        split[3] = strings.Title(split[3])
                   ^
hack/csv-generator.go:68:17: Error return value of `rootCmd.Execute` is not checked (errcheck)
        rootCmd.Execute()
                       ^
hack/csv-generator.go:82:26: Error return value of `rootCmd.MarkFlagRequired` is not checked (errcheck)
        rootCmd.MarkFlagRequired("csv-version")
                                ^
hack/csv-generator.go:83:26: Error return value of `rootCmd.MarkFlagRequired` is not checked (errcheck)
        rootCmd.MarkFlagRequired("namespace")
                                ^
hack/csv-generator.go:84:26: Error return value of `rootCmd.MarkFlagRequired` is not checked (errcheck)
        rootCmd.MarkFlagRequired("operator-image")
                                ^
hack/csv-generator.go:275:30: Error return value of `unstructured.SetNestedSlice` is not checked (errcheck)
                unstructured.SetNestedSlice(r.Object, deployments, "spec", "install", "spec", "deployments")
                                           ^
hack/csv-generator.go:279:30: Error return value of `unstructured.SetNestedSlice` is not checked (errcheck)
                unstructured.SetNestedSlice(r.Object, relatedImages, "spec", "relatedImages")
                                           ^
internal/operands/node-labeller/reconcile_test.go:65:23: Error return value is not checked (errcheck)
                reconcileClusterRole(&request)
                                    ^
internal/operands/node-labeller/reconcile_test.go:66:26: Error return value is not checked (errcheck)
                reconcileServiceAccount(&request)
                                       ^
tests/tests_suite_test.go:372:2: S1038: should use fmt.Printf instead of fmt.Println(fmt.Sprintf(...)) (but don't forget the newline) (gosimple)
        fmt.Println(fmt.Sprintf("timeout set to %d minutes", env.Timeout()))
        ^
tests/tests_suite_test.go:373:2: S1038: should use fmt.Printf instead of fmt.Println(fmt.Sprintf(...)) (but don't forget the newline) (gosimple)
        fmt.Println(fmt.Sprintf("short timeout set to %d minutes", env.ShortTimeout()))
        ^
tests/tests_suite_test.go:377:3: S1038: should use fmt.Printf instead of fmt.Println(fmt.Sprintf(...)) (but don't forget the newline) (gosimple)
                fmt.Println(fmt.Sprintf("TopologyMode set to %s", envTopologyMode))
                ^
internal/operands/template-validator/reconcile_test.go:152:3: ineffectual assignment to reconcileResults (ineffassign)
                reconcileResults, err := operand.Reconcile(&request)
                ^
```

Reply:
ACK